Packs/BeyondTrustEPM/README.md

```diff
@@ -0,0 +1,59 @@
+    - Region: Select or search for the name of the region where your storage bucket resides.
+5. Select the data format: CIM - Common Information Model or ECS - Elastic Common Schema.
+6. Select Server-Side Encryption to encrypt files sent to the S3 bucket using the default AWS encryption key.
+7. Click Validate Settings to test the connection to your storage site.
+8. Click Save Settings.
+ 
+For more information <[Link to the official docs](https://docs.beyondtrust.com/epm-wm/docs/welcome-to-endpoint-privilege-management-for-windows-and-mac)>.
+ 
+### Cortex XSIAM side - Amazon S3 Collector
+To connect Cortex XSIAM to the AWS S3 bucket, follow the below steps.
+1. Navigate to **Settings** -> **Data Sources** -> **Add Data Source**.
+2. If you have already configured an **Amazon S3 Collector**, select the **3 dots**, and then select **+ Add New Instance**. If not, select **+ Add Data Source**, search for "Amazon S3" and then select **Connect**.
+4. Set the following values:
+ 
+    | Parameter    | Value                                                                                                                                           |
+    |:-------------|:------------------------------------------------------------------------------------------------------------------------------------------------|                 
+    | `SQS URL`    | Refer to Configure an Amazon Simple Queue Service (SQS) [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3). |
+    | `Name`       | BeyondTrust EPM Logs.                 |
+    | `AWS Client ID`     |                                                                                                                                   |
+    | `AWS Client Secret`    |                                                                                                                                  |
+    | `Log Type`     | Generic.                                                                                                                                  |
+    | `Log Format`    | Select the relevant log format.                                                                                                                                 |
+    | `Vendor`     | Enter BeyondTrust.                                                                                                                                  |
+    | `Product`    | Enter EPM.                                                                                                                                 |
+    | `Compression`    | Select the desired compression.                                                                                                                                 |
+ 
+For more information, see this [doc](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3).
```

Comment:
For more information, see [Ingest Generic Logs from Amazon S3](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3).